packages/blocks/src/api/raw-handling/test/utils.js

```diff
@@ -7,12 +7,15 @@ import deepFreeze from 'deep-freeze';
  * Internal dependencies
  */
 import { getBlockContentSchema, isPlain } from '../utils';
+import { store as mockStore } from '../../../store';
+import { STORE_NAME as mockStoreName } from '../../../store/constants';
 
 jest.mock( '@wordpress/data', () => {
 	return {
 		select: jest.fn( ( store ) => {
 			switch ( store ) {
-				case 'core/blocks': {
+				case [ mockStoreName ]:
```

Comment:
Quite tricky, but it makes a lot of sense 👍

Reply:
I could be removing the `switch` statement to an `if` if that's better 😅.

Reply:
I was commenting about the whole process of mocking. It is it what it is, we better leave it 😃